lib/components/shopping-cart-widget-component.js

```diff
@@ -10,96 +10,41 @@ export default class ShoppingCartWidgetComponent extends BaseContainer {
 		super( driver, by.className( 'cart-toggle-button' ) );
 	}
 
-	open() {
-		let d = webdriver.promise.defer();
+	async open() {
 		const driver = this.driver;
-		const explicitWait = this.explicitWait;
-
-		this.displayedOpen().then(
-			function cartAlreadyOpen() {
-				d.fulfill( true );
-			},
-			function cartClosed() {
-				driverHelper
-					.clickWhenClickable( driver, by.css( '.cart-toggle-button' ), explicitWait )
-					.then(
-						function success() {
-							d.fulfill( true );
-						},
-						function failure() {
-							d.reject( 'Unable to find Shopping Cart Widget button' );
-						}
-					);
-			}
-		);
-
-		return d.promise;
+		const explicitWait = this.explicitWaitMS;
+
+		if ( ! await this.displayedOpen() ) {
+			return await driverHelper.clickWhenClickable(
+				driver,
+				by.css( '.cart-toggle-button' ),
+				explicitWait
```

Comment:
this explicit wait isn't actually necessary as it's the same as the default value - would you remind removing it? (added before we defaulted the value I believe)

Reply:
Done. Did the same for driver.